hvantk/tests/test_feature_stats.py

```
@@ -4,31 +4,68 @@
+    reason="Hail tests require RUN_HAIL_TESTS=1",
 )
 
+
 def safe_float(val):
     return hl.missing(hl.tfloat64) if val is None else float(val)
 
+
 @pytest.mark.hail
 def test_feature_coverage_stats():
-    hl.init(log='/tmp/hail_test.log', quiet=True)
+    hl.init(log="/tmp/hail_test.log", quiet=True)
```

Ruff (0.14.13)

Probable insecure usage of temporary file or directory: "/tmp/hail_test.log"

(S108)